src/test/resources/project-with-path-pattern/src/main/java/source.java

```diff
@@ -0,0 +1,22 @@
+/*-
+ * #%L
+ * OpenFastTrace Maven Plugin
+ * %%
+ * Copyright (C) 2018 - 2021 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
```

Comment:
The license header distracts from the real intention of this file. Please add an exclusion to the license checker plugin and remove the header.

Reply:
Done.